src/main/resources/containers.json

```diff
@@ -1250,8 +1250,6 @@
     ]
   },
   {
-    "artifact_id": "glassfish",
-    "group_id": "org.glassfish.main.distributions",
     "name": "Arquillian Container GlassFish Managed",
```

Comment:
If that's handled now by chameleon and we have new `model` module to "query" for containers do we still have to keep those entries here?